Sources/LanguageServerProtocol/SupportTypes/Language.swift

```diff
@@ -153,6 +154,7 @@ public extension Language {
   static let shellScript = Language(rawValue: "shellscript")  // Shell Script (Bash)
   static let sql = Language(rawValue: "sql")
   static let swift = Language(rawValue: "swift")
+  static let tutorial = Language(rawValue: "tutorial")
```

Comment:
This is an LSP extension, right? Could you mark it as such and add it to the LSP extensions document?

Reply:
Yes it is. I've added the documentation.